client/branded/src/global-styles/code.scss

```
@@ -1,5 +1,5 @@
 $code-font-family: sfmono-regular, consolas, menlo, dejavu sans mono, monospace;
-$code-font-size: 12px;
+$code-font-size: 0.92582em;
```

How did you get to this value? Shouldn't this be 12/14 (you could even write that as a division in Sass), so it becomes 12px for 14px surrounding font? Or am I missing something?
```suggestion
$code-font-size: (12/14)em;
```